stripe-sms-receipt/functions/send-sms-receipt.js

```diff
@@ -0,0 +1,68 @@
+/**
+ *  Send Stripe SMS Receipt
+ *
+ *  This function shows you how to programmatically send a payment receipt URL to your customer via SMS,
+ *  using a Stripe webhook event as trigger.
+ *
+ *  Pre-requisites:
+ *  - Enable ACCOUNT_SID and AUTH_TOKEN in your functions configuration (https://www.twilio.com/console/functions/configure)
+ *  - Add STRIPE_SECRET_KEY to your environment variables (https://www.twilio.com/console/functions/configure)
+ *  - Add stripe to your NPM package dependencies (https://www.twilio.com/console/functions/configure)
+ */
+
+exports.handler = async function(context, event, callback) {
+  const response = new Twilio.Response();
+  response.appendHeader("Content-Type", "application/json");
+
+  // Uncomment to support CORS.
+  // response.appendHeader('Access-Control-Allow-Origin', '*');
+  // response.appendHeader('Access-Control-Allow-Methods', 'POST, OPTIONS');
+  // response.appendHeader('Access-Control-Allow-Headers', 'Content-Type');
+
+  // Only action charge.succeeded events
+  if (event.type === "charge.succeeded") {
+    // Retrieve the event object from Stripe to verify its validity.
+    // If you're running your own server environment you can verify
+    // the webhook signature instead:
+    // https://stripe.com/docs/webhooks/signatures#verify-official-libraries
+    try {
+      // To use stripe in your Twilio function you need to add it in the dependencies
+      // section of your functions config: https://www.twilio.com/console/functions/configure.
+      const stripe = require("stripe")(context.STRIPE_SECRET_KEY);
+      const stripeEvent = await stripe.events.retrieve(event.id);
+      const charge = await stripe.charges.retrieve(stripeEvent.data.object.id, {
+        expand: ["customer"]
+      });
+      const customer = charge.customer;
+      // Look for phone number on the customer otherwise use phone number from the
+      // billing details of the charge.
+      const cust_phone = customer
```

Comment:
Can you use camelCase variables? Just for consistency between them :) I think we are working on adding a linter but don't have it yet.

Reply:
Thanks for catching 👍